src/commands/dumpDb.ts

```diff
@@ -0,0 +1,17 @@
+import { DumpDbOptions, Registry } from "../types";
+import { DBService } from "../utils";
+
+/**
+ * Dump the database as JSON to STDOUT for a given chain ID
+ * @param options A dict, but essentially just the chainId
+ */
+export async function dumpDb(options: DumpDbOptions) {
+  const { chainId } = options;
+
+  Registry.dump(DBService.getInstance(), chainId.toString())
+    .then((dump) => console.log(dump))
+    .catch((error) => {
+      console.error(error);
```

Comment:
ubernit, always nice to right some context next to the error.
Like console.error('Error dumping DB', error);

Reply:
Done.